code/modules/reagents/reagent_dispenser.dm

```diff
@@ -178,11 +178,11 @@
 
 /obj/structure/reagent_dispensers/fueltank/proc/explode()
 	if (reagents.total_volume > 500)
-		explosion(10, 3)
+		explosion(10, 2)
```

Comment:
modify the range not the power, fueltank is the most powerful explosive because its so costly

drop the range by 3-4 tiles here instead, maybe 2 less for the other sizes

Reply:
Oh, shit, edited the wrong integers actually. Will phix.

Reply:
you didnt fix this bit yet